test/js/base/config.js

```diff
@@ -64,4 +64,97 @@ describe("Config", function() {
             assert.strictEqual(instance.hasInitialsRadius(), true);
         });
     });
+
+    describe("#initialsConfig()", function() {

```

Comment:
This was ported from by https://github.com/ripe-tech/ripe-compose/blob/08b211a1c7968cdb655ce1c4ffbe14ecbb15dd80/src/ripe_compose/test/util/configurator.py#L548 correct?

Reply:
Yes